activation/validation.go

```diff
@@ -365,20 +364,6 @@ func (v *Validator) VerifyChain(ctx context.Context, id, goldenATXID types.ATXID
 	return v.verifyChainWithOpts(ctx, id, goldenATXID, options)
 }
 
-func (v *Validator) getNipost(ctx context.Context, id types.ATXID) (*types.NIPost, error) {
-	var blob sql.Blob
-	if err := atxs.LoadBlob(ctx, v.db, id.Bytes(), &blob); err != nil {
-		return nil, fmt.Errorf("getting blob for %s: %w", id, err)
-	}
-
-	// TODO: decide about version based on publish epoch
-	var atx wire.ActivationTxV1
-	if err := codec.Decode(blob.Bytes, &atx); err != nil {
-		return nil, fmt.Errorf("decoding ATX blob: %w", err)
-	}
-	return wire.NiPostFromWireV1(atx.NIPost), nil
-}
```

Comment:
I'd prefer having it here or in the `activation.Certifier` instead of in the `sql/atxs` package.

Reply:
Moved to activation/certifier.go